test/npm.test.ts

```diff
@@ -1,10 +1,23 @@
-import npm from "~/npm";
-
 describe("npm", () => {
-  test("adds npm and semantic-release/git plugins", () => {
+  afterEach(() => {
+    delete process.env.GITHUB_REF;
+  });
+
+  test("adds npm and semantic-release/git plugins", async () => {
+    const npm = await import("~/npm");
```

Comment:
I think we need to add `process.env.GITHUB_REF="main";` here for the test to work 💡